portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,31 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
+    <head>
+        <meta charset="UTF-8">
```

Comment:
nit: 2 spaces indent.

Reply:
2 spaces indent is in #2 - I'll try to keep stuff more organized next time, this one page design made it kind of hard to make things not dependent on each other...

Reply:
Actually I think it might be easier to solve it here. This will make #2 simpler. Currently the reformatting in #2 distracts me from the core part.